ambry-api/src/main/java/com.github.ambry/router/GetBlobOptions.java

```diff
@@ -28,6 +28,8 @@
   private final ByteRange range;
   // Flag indicating whether to return the raw blob payload without deserialization.
   private final boolean rawMode;
+  private final long blobSegment;
+  public static long NO_BLOB_SEGMENT_IDX_SPECIFIED = -1;
```

Comment:
Can leave out _IDX since the variable is called blobSegment not blobSegmentIndex.

Reply:
changing `blobSegment` to `blobSegmentIdx`